spec/index.html

```diff
@@ -816,6 +816,27 @@ <h3>Content Negotiation of RDF Datasets</h3>
       the consumer is expected to use the <a data-lt="rdf dataset">RDF dataset's</a> default graph.</p>
 
     </section>
+
+    <section id="section-dataset-quad" class="informative">
+      <h3>Dataset as a set of quads</h3>
+
+      <p>A <dfn>quad</dfn> is a conceptual projection of a <a>triple</a> with
+        either the <a>default graph</a> or a <a>named graph</a>.
+        A <a>quad</a> is the generalization of a <a>triple</a>
+        along with an optional <a>graph name</a>,
+        indicating a named graph containing that <a>triple</a>.
+      </p>
+
+      <p>A <a>quad</a> can be represented as a tuple composed of <a>subject</a>, <a>predicate</a>, <a>object</a>,
+        and an optional <a>graph name</a>.</p>
+
+      <p>An <a>RDF dataset</a> can informally be considered to be a set of <a>quads</a>.</p>
+
+      <p class="note">Although a <a>quad</a> in the <a>default graph</a> shares
```

Comment:
A quad is not "in" a graph.  A graph is a set of triples.

The text should say how quads relate to the graph of a dataset.

If we follow #5 and a quad is "triple+graph name":
```
<p>An <a>RDF dataset</a> can be considered to be a set of <a>quads</a>
where the quads no graph name supply the triples of the default graph
and the quads with the same graph name, supply the triples of the named graph with
that name.</p>
```

Reply:
Punctuation and language tweaks, and addition of a missing word...
```
<p>An <a>RDF dataset</a> can be considered to be a set of <a>quads</a>
where quads with no graph name supply the triples of the default graph,
and quads with the same graph name supply the triples of the named graph with
that name.</p>
```